docs/settings.md

```diff
@@ -83,7 +83,7 @@ connecting IPs in the `forwarded-allow-ips` configuration.
 * `--date-header` / `--no-date-header` - Enable/Disable default `Date` header.
 
 !!! note
-    The `--no-server-header` flag doesn't have effect on the WebSockets implementations.
+    The `--no-server-header` and `--no-date-header` flags doesn't have effect on the WebSockets implementations.
```

Comment:
Could these fixes be their own PR? They don't seem directly related to deployments, I'm guessing your just noticed them as you were working on the new docs, right?

Reply:
Yes, this was supposed to be in another PR. I'll fix it.